<launch>
  <!-- Launch file for gpio_msgs (not a complete application) -->
  <node pkg="gpio_msgs" type="publish_gpio_values.py" name="sb_publish_gpio_values" output="log"/>
  <node pkg="gpio_msgs" type="serve_gpio_get.py" name="sb_serve_gpio_get" output="log"/>
  <node pkg="gpio_msgs" type="serve_gpio_set.py" name="sb_serve_gpio_set" output="log"/>
  <node pkg="gpio_msgs" type="serve_gpio_info.py" name="sb_serve_gpio_info" output="log"/>
</launch>
